core/i18n/src/test/ITwinLocalization.test.ts

```diff
@@ -146,7 +508,583 @@ describe("ITwinLocalization", () => {
+
+      it("first level fails", () => {
+        assert.equal(localization.getEnglishString("Nonexisting", "FirstTrivial"), "FirstTrivial");
+        assert.equal(localization.getEnglishString("Nonexisting", "MissingKeyString"), "MissingKeyString");
+      });
+
+      it("second level fails", () => {
+        assert.equal(localization.getEnglishString("Nonexisting", "SecondTrivial.Test1"), "SecondTrivial.Test1");
+        assert.equal(localization.getEnglishString("Nonexisting", "Missing.String"), "Missing.String");
+      });
+
+      it("empty string namespace is treated as default namespace", () => {
+        assert.equal(localization.getEnglishString("", "FirstTrivial"), "First level string (default)");
+        assert.equal(localization.getEnglishString("", "SecondTrivial.Test1"), "Second level string 1 (default)");
+      });
+    });
+
+    // Test a few options to make sure they get passed through correctly
+    describe("With Options", () => {
+
+      it("returnDetails throws error", () => {
+        assert.throws(() => { localization.getEnglishString("Default", "X", { returnDetails: true }); }, "Translation key must map to a string, but the given options will result in an object");
+      });
+
+      it("returnObjects throws error", () => {
+        assert.throws(() => { localization.getEnglishString("Default", "X", { returnObjects: true }); }, "Translation key must map to a string, but the given options will result in an object");
+      });
+
+      it("returnDetails and returnObjects throws error", () => {
+        assert.throws(() => { localization.getEnglishString("Default", "X", { returnDetails: true, returnObjects: true }); }, "Translation key must map to a string, but the given options will result in an object");
+      });
+
+      it("default value", () => {
+        assert.equal(localization.getEnglishString("Default", "Missing", { defaultValue: "default" }), "default");
+      });
+
+      it("english takes priority over fallback language", () => {
+        assert.equal(germanLocalization.getEnglishString("Default", "OnlyEnglish", { fallbackLng: "de" }), "Hello");
+      });
+
+      // White box test
+      /* TODO:
+       * This functionality doesn't make sense. It should work the opposite like how getLocalizedStringWithNamespace() does.
+       * getEnglishString() shouldn't take in a namespace. The 'namespace' parameter should be removed.
+       * A getEnglishStringWithNamespace() method could be created.
+       * This is a breaking change, so is left as a to-do until discussed and approved.
+       */
+      it.skip("given namespace DOES NOT override namespace translation option", () => {
+        assert.equal(localization.getEnglishString("Default", "FirstTrivial", { ns: "Test" }), "First level string (test)");
+        assert.equal(localization.getEnglishString("Default", "FirstSubstitution1", { str: "CUSTOM1", ns: "Test" }), "First level CUSTOM1 (test)");
+      });
+    });
```

Comment:
@calebmshafer or @wgoehrig What do you think about my comment here? `getEnglishString()` takes in a `namespace` parameter which doesn't really make sense. I'd think it should be like `getLocalizedString()` and `getLocalizedStringWithNamespace()` where there is another function that handles providing a namespace parameter.

For `getLocalizedStringWithNamespace()`, the passed in namespace is used no matter what i18next options are passed in (see this [test](https://github.com/iTwin/itwinjs-core/blob/552c2fa584673f170f8e2a7dbca89d49b36a1f34/core/i18n/src/test/ITwinLocalization.test.ts#L747)). For `getEnglishString()`, this is not the case.

I understand if we don't want to make these changes right now, but should I leave this test as skipped, comment it out, or just delete it?

Reply:
I think we should iron out the differences between the english and non-english methods however that cannot be done through a breaking change.

Adding a new method would be fine, is there anywhere it will actually be used in the code?

Reply:
Yea, even though it's not the most ideal the most important part is that we don't break the API as it is today.